static/js/reactions.js

```diff
@@ -109,6 +115,51 @@ $(document).on('click', '.reaction-popover-reaction', function () {
 $(document).on('input', '.reaction-popover-filter', filter_emojis);
 $(document).on('keydown', '.reaction-popover-filter', maybe_select_emoji);
 
+exports.reaction_navigate = function (e, event_name) {
+    var first_emoji = get_emoji_from_index(0);
+    var selected_emoji = $(".reaction-popover-reaction").filter(":focus")[0];
```

Comment:
We may want a small helper for this, since we use it in a few places, and since jQuery makes it a bit fiddly to find the selected emoji.